### ISSUE TITLE
BUG - Sev 3 - iOS - Alert component heading not announced with VoiceOver

### ISSUE THREAD

**Issue by oddballdave** [department-of-veterans-affairs/va-mobile-library#630]
## What happened?

VoiceOver is not correctly announcing the heading of non-expandable alerts on iOS.

## Specs:

- Device: iPhone
- OS: iOS
- User Account: N/A
- Accessibility State: VoiceOver

## Steps to Reproduce

Using VoiceOver on iOS, select the heading of a non-expandable Alert component. It should speak out the heading name and the role of "heading".

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low 
- Frequency: High 

- 3 - Low

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
